pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -9,12 +9,37 @@ import (
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/util/sets"
 
-	"github.com/openshift/origin/pkg/api"
 	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
 	authorizationapiv1 "github.com/openshift/origin/pkg/authorization/api/v1"
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
 	imageapi "github.com/openshift/origin/pkg/image/api"
+	oauthapi "github.com/openshift/origin/pkg/oauth/api"
 	projectapi "github.com/openshift/origin/pkg/project/api"
 	routeapi "github.com/openshift/origin/pkg/route/api"
+	sdnapi "github.com/openshift/origin/pkg/sdn/api"
+	templateapi "github.com/openshift/origin/pkg/template/api"
+	userapi "github.com/openshift/origin/pkg/user/api"
+)
+
+var (
+	readWrite = []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"}
+	read      = []string{"get", "list", "watch"}
+
+	kapiGroup        = kapi.GroupName
+	autoscalingGroup = autoscaling.GroupName
+	batchGroup       = batch.GroupName
+	extensionsGroup  = extensions.GroupName
+	authzGroup       = authorizationapi.GroupName
```

Comment:
> do you want these to silently change from "" to "authorization.openshift.io" when we groupify?

Do I want them to be right?  Yeah.  It won't be silent since we have a unit test watching for strict equality.  When it happens, I suspect we'll have policy accepting both for some time.